src/codegen/compile.ml

```diff
@@ -4392,8 +4392,15 @@ module Serialization = struct
       compile_unboxed_const 0l ^^ set_ref_size ^^
 
       let inc_data_size code =
+        let (set_sum, get_sum) = new_local env "sum" in
```

Comment:
OCaml doesn't mandate the parens.
```suggestion
        let set_sum, get_sum = new_local env "sum" in
```

Reply:
When in Rome - do as the Romans - most of the other code make the tuple explicit...